### PR TITLE
Terrain layer

### PR DESCRIPTION
**Pull request by nilscb**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (hkfb) [inline package.json:1]: Is this file needed?

File: react/package.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
         "lint": "eslint *.js *.json \"src/**/*.+(ts|tsx|js|jsx|json|css)\"",
         "validate": "npm run typecheck && npm run lint",
         "setup-deckgl-types": "cd ./node_modules/@danmarshall/deckgl-typings/ && indefinitely-typed --folder deck.gl__aggregation-layers --folder deck.gl__core --folder deck.gl__extensions --folder deck.gl__geo-layers --folder deck.gl__google-maps --folder deck.gl__json --folder deck.gl__layers --folder deck.gl__mapbox --folder deck.gl__mesh-layers --folder deck.gl__react --folder deck.gl --folder luma.gl__constants --folder luma.gl__core --folder luma.gl__gltools --folder luma.gl__webgl-state-tracker --folder luma.gl__webgl --folder math.gl__core --folder math.gl",
-        "storybook": "start-storybook -p 6006",
+        "storybook": "start-storybook -p 6006 -s public",
```
Comment (hkfb) [inline react/package.json:33]: Doesn't look like you included the depth map file. Perhaps this should point to the 'example-data' directory.

**Review by hkfb — CHANGES_REQUESTED** [review]

File: react/package.json [hunk 3]
```diff
@@ -50,6 +50,9 @@
         "@equinor/eds-core-react": "^0.12.1",
         "@equinor/eds-icons": "^0.6.2",
         "@equinor/videx-wellog": "^0.4.5",
+        "@loaders.gl/core": "^2.3.13",
+        "@loaders.gl/images": "^2.3.13",
+        "@loaders.gl/terrain": "^3.0.13",
```
Comment (hkfb) [inline react/package.json:55]: @nilscb Are these needed explicitly? They are installed automatically on my system

File: react/src/lib/components/DeckGLMap/layers/terrain/map3DLayer.ts [hunk 4]
```diff
@@ -0,0 +1,73 @@
+import { CompositeLayer } from "@deck.gl/core";
+import TerrainMapLayer, {
+    TerrainMapLayerProps,
+    TerrainMapLayerData,
+} from "./terrainMapLayer";
+import { ExtendedLayerProps } from "../utils/layerTools";
+import { layersDefaultProps } from "../layersDefaultProps";
+import { TerrainLoader } from "@loaders.gl/terrain";
```
Comment (hkfb) [inline react/src/lib/components/DeckGLMap/layers/terrain/map3DLayer.ts:8]: Got a typescript error from this line. Replacing with 
```
const terrain = require("@loaders.gl/terrain");
```
seems to work.

@shadab-skhan @shruthirai @Vladimir-Kokin any of you get an error?

**Review by hkfb — CHANGES_REQUESTED** [review]

File: react/src/lib/components/DeckGLMap/layers/terrain/map3DLayer.ts (hunk 4 above)
Comment (nilscb) [reply react/src/lib/components/DeckGLMap/layers/terrain/map3DLayer.ts:8]: I think you have to install it.. npm install loaders.gl/terrain

**Review by hkfb — APPROVED** [review]
